### ISSUE TITLE
Save menu really confusing for shared files

### ISSUE THREAD

**Issue by shriram** [brownplt/code.pyret.org#205]
I get a shared file. I make some changes. I'd like to save them.

The Save entry is greyed out. It's natural to assume it's auto-saved. (One teacher actually thought their work was being auto-saved all along when it wasn't being saved _at all_! Heck, there's even a filename showing…)

Suppose I'm skeptical and I make some changes to the buffer, then go to the menu. Save is _still_ greyed out. I can see _Save as_, but I don't want to save as any other name — I'm happy with the name I already had (the one visible at the top).

In other words, this is all pretty confusing.

When you have a shared file, you should have neither Save nor Save as, but rather something like "Save a personal copy".

**Comment by schanzer** [comment]
@shriram are you happy with this fix?

**Comment by schanzer** [comment]
@shriram poke?
